packages/grzctl/src/grzctl/commands/pruefbericht.py

```diff
@@ -135,6 +136,10 @@ def pruefbericht(config_file, submission_dir, output_json, failed, token, dry_ru
     if config.pruefbericht.client_secret is None:
         raise ValueError("pruefbericht.client_secret must be provided to submit Prüfberichte")
 
+    if metadata.submission.tan_g == REDACTED_TAN and not "test" in config.pruefbericht.api_base_url:
```

Comment:
Not sure whether we really need the `not "test" in config.pruefbericht.api_base_url` check (may be useful for submitting to the test endpoint with a redacted tanG)

Reply:
Hmm, checking for "test" in the URL seems a little hacky; what about a command-line flag to allow redacted TAN submission? With default to error?

Reply:
I agree, let's do that instead.